src/components/CardComponent.js

```diff
@@ -6,34 +6,61 @@ import CardContent from '@material-ui/core/CardContent'
 import CardMedia from '@material-ui/core/CardMedia'
 import Grid from '@material-ui/core/Grid'
 import Typography from '@material-ui/core/Typography'
+import JobModal from './JobModal';
+import JobModalNew from './JobModalNew';
 
+class CardComponent extends React.Component {
+
+
+  constructor(props) {
+    super(props)
+    this.state = { isModalOpen: false }
+  }
+
+  openJobModal() {

```

Comment:
you don't need both of these - you can just do `toggleJobModal () { this.setState({ isOpen: !this.state.isOpen }`